File: wind-recruitment-ui/src/app/services/models/candidacy-response.ts
/* tslint:disable */
/* eslint-disable */
import { Candidate } from '../models/candidate';
import { Topic } from '../models/topic';
export interface CandidacyResponse {
  candidate?: Candidate;
  createdAt?: string;
  file?: Array<string>;
  id?: number;
  status?: string;
  topic?: Topic;
}

File: wind-recruitment-api/src/main/java/com/wind/windrecruitmentapi/utils/candidacies/CandidacyResponse.java
package com.wind.windrecruitmentapi.utils.candidacies;

import com.wind.windrecruitmentapi.entities.Candidate;
import com.wind.windrecruitmentapi.entities.Topic;
import com.wind.windrecruitmentapi.utils.topics.TopicResponse;

public record CandidacyResponse(
        Integer id,
        String candidate_full_name,
        TopicResponse topic,
//        byte[] file,
        String file_url,
        String file_name,
        String createdAt,
        String status

) {
}
